.github/workflows/simple.yaml

```diff
@@ -0,0 +1,134 @@
+---
+name: Simple workflow
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    branches:
+      - main
+  push:
+    branches:
+      - main
+      - setup-workflows
+
+env:
+  # additional flags for `R CMD CHECK`
+  EXTRA_CHECK_FLAGS: "--no-manual --no-vignettes"
+  # additional environmental vars for `R CMD CHECK`
+  # see manual here: https://rstudio.github.io/r-manuals/r-ints/Tools.html
+  _R_CHECK_TESTS_NLINES_: 0
+  _R_CHECK_VIGNETTES_NLINES_: 0
+  # When asked to, please add the steps from the list above:
+  ENABLED_STEPS: |
+    checkout_repository
+#    setup_r
+#    setup_dependencies
+#    get_package_name
+#    build_r_package
+#    run_r_cmd_check
+#    upload_artifact
+
+jobs:
+  # First job.
+  hello-world-from-r:
+    name: Hello, world!
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup R 🔧
+        uses: r-lib/actions/setup-r@v2
+
+      - name: Hello world! 💬
+        run: cat(paste0("Hello ", Sys.getenv("NAME"), "! 👋"))
+        shell: Rscript {0}
+        env:
+          NAME: "Your name"
+
+  # Second job - by default jobs will run in parallel, unless some dependencies
+  # are introduced between them.
+  build-and-check:
+    name: Build R package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo 🛎
+        # We're using an official GitHub Action.
+        uses: actions/checkout@v4.1.1
+        if: contains(env.ENABLED_STEPS, 'checkout_repository')
+        with:
+          # Action inputs (arguments/parameters).
+          # In this case we clone the repository to the directory called the same
+          # as the repository name.
+          path: ${{ github.event.repository.name }}
+
+      - name: Setup R 🔧
+        if: contains(env.ENABLED_STEPS, 'setup_r')
+        # We're using a third-party GitHub Action.
+        uses: r-lib/actions/setup-r@v2
+
+      - name: Setup R dependencies 🔧
+        # We're using a third-party GitHub Action.
+        uses: r-lib/actions/setup-r-dependencies@v2
+        if: contains(env.ENABLED_STEPS, 'setup_dependencies')
+        with:
+          extra-packages: any::rcmdcheck
```

Comment:
I can see that you are using `R CMD` way of running checks which is perfectly fine. Now I'm curious whether this is needed at all. Can we make a test run without it?

Reply:
Do you have any suggestion how we could do that?

Reply:
just remove this line and see what happens :)

Reply:
Ah, okay, [looks like it works](https://github.com/user-workshop-cicd/r.package.example/actions/runs/9464158221/job/26070872953), there's an expected failure.

Reply:
yup - it looks good. let's leave it removed then